src/docfx/localization/LocalizationUtility.cs

```diff
@@ -193,6 +193,29 @@ public static bool TryRemoveLocale(string name, out string nameWithoutLocale, ou
             return false;
         }
 
+        public static bool TryTrimPathLeadingLocale(PathString path, out PathString pathWithoutLocale, out string locale)
+        {
+            pathWithoutLocale = default;
+            locale = null;
+            if (string.IsNullOrEmpty(path))
+            {
+                return false;
+            }
+
+            var isAbsolute = path.Value[0] == '/';
```

Comment:
when will path start with `/`?

Reply:
Currently it won't, will limit this functionality inside redirection provider